### DISCUSSION TITLE
chore: Improve caprover installation experience

### DISCUSSION THREAD

**Discussion by vishnu-narayanan** [chatwoot/chatwoot#8463] [Self hosted]
- [ ] Check if prefilling FRONTEND_URL is possible
- [ ] Bundle SMTP server to have email notifications/password resets work out of the box
- [ ] Ability to use local storage option in caprover

**Comment by vishnu-narayanan** [comment]
https://github.com/chatwoot/charts/issues/18

**Comment by uhlhosting** [comment]
any progress here? Its a pitty all data is lost upon making changes to env variables in caprover.

**Comment by mredodos** [comment]
some update?

**Comment by pranavrajs** [comment]
We are moving this to discussions as the Caprover installation is no longer officially supported by the team. It is maintained by the community.
